dashboard/config/scripts/frequency_analysis.script

```diff
@@ -0,0 +1,6 @@
+title 'Frequency Analysis Widget'
+description 'Use the Frequency Analysis Widget to analyze letter frequencies in an input text compared to expected frequencies for standard English. Use that comparison to explore cracking Shift and Substitution Ciphers.'
+description_short 'Frequency Analysis Widget'
+
+stage 'Frequency Analysis'
+level 'Frequency Analysis'
```

Comment:
Nit: I think this file needs to be named `frequency.script` if you want it to show up as `/s/frequency/stage/1/puzzle/1`.  It shows up as `/s/frequency_analysis/stage/1/puzzle/1` for me after seed.